interpreter/script/run.ml

```diff
@@ -458,11 +464,20 @@ let run_assertion ass =
   | AssertInvalid (def, re) ->
     trace "Asserting invalid...";
     (match
-      let m, cs = run_definition def in
-      Valid.check_module_with_custom (m, cs)
```

Comment:
I think this should be left as is -- we also want to check that we do _not_ get custom errors here.